package.json

```diff
@@ -18,6 +18,7 @@
     "lint:fix": "yarn lint --fix",
     "lint-docs": "prettier '**/*.md' --check",
     "lint-docs:fix": "prettier '**/*.md' --write",
+    "lint-dashboards": "scripts/validate-grafana-dashboards.sh",
```

Comment:
I think we should not mix up validation and linting, the validate script does other things like git diff which is not required if you lint locally.

Would recommend the following instead

`"lint-dashboards": "node scripts/lint-grafana-dashboards.mjs ./dashboards",`

Reply:
i like the idea of being able to run the same command that the CI runs so I can see if it will pass/fail before pushing the commit.  I suppose we can change the script command name?

Reply:
We have a bunch of those scripts in `./scripts` folder
- [assert_eslintrc_sorted.mjs](https://github.com/ChainSafe/lodestar/blob/unstable/scripts/assert_eslintrc_sorted.mjs)
- [assert_exports.mjs](https://github.com/ChainSafe/lodestar/blob/unstable/scripts/assert_exports.mjs)

Reply:
Would say lint should lint and not validate but it is a minor detail so no big deal.